Added target_bulk_bytes to the docs for logstash-output plugin (#7869)
opensearch-project/documentation-website: _tools/logstash/ship-to-opensearch.md

### Required Parameters

- `hosts` (array of string): The `AmazonOpensearchService` domain endpoint and port number.
- `auth_type` (JSON object): The authentication settings.
    - `type` (string): "aws_iam".
    - `aws_access_key_id` (string): AWS access key.
    - `aws_secret_access_key` (string): AWS secret access key.
    - `region` (string, :default => "us-east-1"): The region in which the domain is located.
- port (string): AmazonOpensearchService listens on port 443 for `HTTPS`.
- protocol (string): The protocol used to connect. For `AmazonOpensearchService`, the protocol is `https`.

### Optional Parameters

- `template` (path): You can set the path to your own template here. If no template is specified, the plugin uses the default template.

> - `template_name` (string, default => "logstash"): Defines how the template is named inside OpenSearch.

Reviewer: ```suggestion
- `template_name` (string, default => `logstash`): Defines how the template is named inside OpenSearch.
```

> - `service_name` (string, default => "es"): Defines the service name to be used for `aws_iam` authentication.

Reviewer: ```suggestion
- `service_name` (string): Defines the service name to be used for `aws_iam` authentication.
```

> - `legacy_template` Bboolean, default => `true`): Selects the OpenSearch template API. When `true`, uses legacy templates derived from the `_template` API. When `false`, uses the `index_template` API.

Reviewer: ```suggestion
- `legacy_template` (Boolean, default => `true`): Selects the OpenSearch template API. When `true`, uses legacy templates derived from the `_template` API. When `false`, uses the `index_template` API.
```

> - `legacy_template` (boolean, default => `true`): Selects the OpenSearch template API. When `true`, uses legacy templates via the _template API. When `false`, uses composable templates via the _index_template API.

Reviewer: ```suggestion
- `legacy_template` Bboolean, default => `true`): Selects the OpenSearch template API. When `true`, uses legacy templates via the `_template` API. When `false`, uses composable templates.
```

- `default_server_major_version` (number): The OpenSearch server major version to use when it's not available from the OpenSearch root URL. If not set, the plugin throws an exception when the version can't be fetched.
- `target_bulk_bytes` (number): The maximum number of bytes in the buffer. When the maximum is reached, Logstash will flush the data to OpenSearch. This is useful when the bulk requests are too large for the OpenSearch cluster and the cluster returns a `429` error.

### Credential resolution logic

The following list provides details on the credential resolution logic:

- A user passes `aws_access_key_id` and `aws_secret_access_key` in the configuration.

> - Environment variables, such `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are recommended since they are recognized by all the AWS SDKs and CLIs except for `.NET`. You can also use `AWS_ACCESS_KEY` and `AWS_SECRET_KEY` which are recgonized by the Java SDK.

Reviewer: ```suggestion
- Environment variables, such `AWS_ACCESS_KEY_ID` and `AWS_SECRET_ACCESS_KEY` are recommended since they are recognized by all the AWS SDKs and CLIs except for `.NET`. You can also use `AWS_ACCESS_KEY` and `AWS_SECRET_KEY` which are recognized by the Java SDK.
```

- service_name (string, default => "es") - Defines the service name to be used for `aws_iam` authentication.
- legacy_template (boolean, default => true) - Selects the OpenSearch template API. When `true`, uses legacy templates via the _template API. When `false`, uses composable templates via the _index_template API.
- default_server_major_version (number) - The OpenSearch server major version to use when it's not available from the OpenSearch root URL. If not set, the plugin throws an exception when the version can't be fetched.

> - target_bulk_bytes (number) - The maximum number of bytes in the buffer. When the maximum is reached, Logstash will flush the data to OpenSearch. This is useful for when the bulk requests are too large for the OpenSearch cluster and the cluster returns a `429` error.

Reviewer 2: ```suggestion
- target_bulk_bytes (number) - The maximum number of bytes in the buffer. When the maximum is reached, Logstash will flush the data to OpenSearch. This is useful when the bulk requests are too large for the OpenSearch cluster and the cluster returns a `429` error.
```

> - target_bulk_bytes (number) - The maximum number of bytes in the buffer. When the maximum is reached, Logstash will flush the data to OpenSearch. This is useful for when the bulk requests are too large for the OpenSearch cluster and the cluster returns a `429` error.

Reviewer 2: Rather than a hyphen separating the term from the description, they should all be either en dashes (--) or colons.

Reviewer: I'll go ahead and fix this really fast before we merge.

> - target_bulk_bytes (number) - The maximum number of bytes in the buffer. When the maximum is reached Logstash will flush the data to Opensearch. This is useful for when the bulk requests are too large for the OpenSearch cluster and you get 429 errors from the cluster.

Reviewer: ```suggestion
- target_bulk_bytes (number) - The maximum number of bytes in the buffer. When the maximum is reached, Logstash will flush the data to OpenSearch. This is useful for when the bulk requests are too large for the OpenSearch cluster and the cluster returns a `429` error.
```
